packages/firebase_ui_auth/lib/src/screens/display_name_screen.dart

```diff
@@ -0,0 +1,180 @@
+// Copyright 2022, the Chromium project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+import 'package:firebase_auth/firebase_auth.dart';
+import 'package:firebase_ui_auth/firebase_ui_auth.dart';
+import 'package:firebase_ui_localizations/firebase_ui_localizations.dart';
+import 'package:firebase_ui_shared/firebase_ui_shared.dart';
+import 'package:flutter/material.dart' hide Title;
+import 'package:flutter/scheduler.dart';
+
+import '../widgets/editable_sign_user_display_name.dart';
+import 'internal/responsive_page.dart';
+
+/// An action that is being called when sign up was successfully.
+class DisplayNameAction extends FirebaseUIAction {
```

Comment:
I'm not sure what this action does and the doc comment is confusing. Why is it called after sign up but called `DisplayNameAction`?